src/main/java/seedu/address/model/expenses/ExpenseType.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.model.expenses;
+
+/**
+ * Represents all the different types of expenses available at the hotel.
+ */
+public enum ExpenseType {
```

Comment:
Suggest changing `enum` to `Class` , and use a `Map` field to track the expense types.